### PR TITLE
pod : optimization Some debian package manager tweaks

### LINKED ISSUE #970: Optimization in Install of debian package
**Which feature do you think can be improved?**

While installing any package with apt/apt-get , it installs a lot of other recommended packages , 
which may or may not , ever used or needed.

**How can it be improved?**

By passing "--no-install-recommends" option, the user lets apt-get know not to consider recommended packages as a dependency to install.

This results in smaller downloads and installation of packages .

Refer to blog at [Ubuntu Blog](https://ubuntu.com/blog/we-reduced-our-docker-images-by-60-with-no-install-recommends) .

 what i have done in #969 

**Before raising this feature request**

Have you looked at the [limitations document](https://github.com/kata-containers/documentation/blob/master/Limitations.md)?

yes

### PR DESCRIPTION
**Pull request by Rajpratik71**
By default, Ubuntu or Debian based "apt" or "apt-get" system installs recommended but not suggested packages .

By passing "--no-install-recommends" option, the user lets apt-get know not to consider recommended packages as a dependency to install.

This results in smaller downloads and installation of packages .

Refer to blog at [Ubuntu Blog](https://ubuntu.com/blog/we-reduced-our-docker-images-by-60-with-no-install-recommends) .

Fixes #970

Signed-off-by: Pratik Raj <rajpratik71@gmail.com>

### CODE REVIEW

**Comment by grahamwhaley** [comment]
/test
Ooh, interesting @Rajpratik71 - any reduction in downloads to save both space and time are **most welcome** :-).
I think you'll need to open an Issue to put a few details in (a copy of the text you have above is fine, as that is nicely detailed, thank you!) - and then you'll need a 'Fixes' reference in at least one commit message - see our [contributing guide](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md) for more details. The CI is going to moan I think...
/cc @chavafg @jcvenegas @GabyCT

**Comment by devimc** [comment]
thanks @Rajpratik71  please fix your commit message, https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format

File: static-build/qemu-virtiofs/Dockerfile [hunk 1]
```diff
@@ -8,11 +8,13 @@ ARG PREFIX
...
 WORKDIR /root/qemu-virtiofs
 RUN apt-get update && apt-get upgrade -y
-RUN apt-get install -y \
+RUN apt-get --no-install-recommends install -y \
+		apt-utils \
 	    autoconf \
 	    automake \
 	    bc \
 	    bison \
+		ca-certificates \
```
Comment (GabyCT) [inline static-build/qemu-virtiofs/Dockerfile]: I think that dependencies should be aligned

Comment (Rajpratik71) [reply static-build/qemu-virtiofs/Dockerfile]: done

**Comment by Rajpratik71** [comment]
> thanks @Rajpratik71 please fix your commit message, https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format

changed . take a look

**Comment by GabyCT** [comment]
@Rajpratik71 it should be 
```
Fixes #970 
```

**Comment by jodh-intel** [comment]
@Rajpratik71 - please can you update your commits to include the initial one-line header with the "subsystem" as explained here:

- https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format

**Comment by Rajpratik71** [comment]
> @Rajpratik71 - please can you update your commits to include the initial one-line header with the "subsystem" as explained here:
> 
> * https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format

done as requested

**Pull request merged**